sonar/modules/documents/dojson/contrib/marc21tojson/model.py

```diff
@@ -848,3 +848,22 @@ def marc21_to_files(self, key, value):
     }
 
     return data
+
+
+@marc21tojson.over('otherEdition', '^775..')
+@utils.for_each_value
+@utils.ignore_value
+def marc21_to_other_edition(self, key, value):
+    """Get other edition."""
+    electronic_locator = value.get('o')
+    public_note = value.get('g')
+
+    if not electronic_locator or not public_note:
```

Comment:
Does ignore_value manage with this? If yes this is useless.

Reply:
ignore_value checks if returned value is None and not the value for all the properties. In addition, in this case, I want to force to have both properties defined in order to store otherEdition.